openlibrary/plugins/openlibrary/js/cover_add_manage.js

```diff
@@ -0,0 +1,58 @@
+export default function(){
+    // page may not be loaded via iframe
+    if (parent && parent.closeThrobber) {
+        parent.closeThrobber();
+    }
+    $('#form.addcover-form').submit(function(event) {
```

Comment:
nitpick: I know this is just the code in openlibrary/templates/covers/add.html but while here would be nice to make this on('submit', so it's clearer its wiring up an event.